components/Navigation.tsx

```diff
@@ -27,174 +25,304 @@ type subroutes = subroute[];
+          },
+          {
+            type: 'route',
+            title: 'Properties',
+            path: '/workspace/tracking-plan/properties',
+          },
+          {
+            type: 'route',
+            title: 'Metrics',
+            path: '/workspace/tracking-plan/metrics',
+          },
+          {
+            type: 'route',
+            title: 'Publishing',
+            path: '/workspace/tracking-plan/publishing',
+          },
+          {
+            type: 'route',
+            title: 'Issue Reporter',
+            path: '/workspace/tracking-plan/issue-reporter',
+          },
+        ],
       },
       { type: 'route', title: 'Connections', path: '/workspace/connections' },
       { type: 'route', title: 'Implement', path: '/workspace/implement' },
-      {
-        type: 'route',
-        title: 'Publishing',
-        path: '/workspace/publishing',
-      },
+      { type: 'route', title: 'Inspector', path: '/workspace/inspector' },
     ],
   },
   {
-    title: 'Developer Tools',
-    path: '/commands',
-    iconName: 'toolbox',
+    title: 'Implementation',
+    path: '/implementation/devs-101',
+    iconName: 'implementation',
     subroutes: [
       {
         type: 'route',
-        title: 'Command line tool',
-        path: '/commands',
+        title: 'Avo 101 for developers',
+        path: '/implementation/devs-101',
+      },
+      {
+        type: 'group',
+        group: 'Getting started guides',
       },
       {
         type: 'route',
-        title: 'Programming languages',
-        path: '/languages',
+        title: 'Getting started with Avo Functions',
+        path: '/implementation/start-using-avo-functions',
       },
       {
         type: 'route',
-        title: 'Analytics tools',
-        path: '/analytics',
+        title: 'Getting started to receive data with a Custom Destination',
+        path: '/implementation/start-custom-destination',
       },
       {
         type: 'route',
-        title: 'Custom destinations',
-        path: '/custom-destinations',
+        title: 'Getting started with the Avo Visual Debuggers',
+        path: '/implementation/start-using-visual-debugger',
       },
       {
         type: 'route',
-        title: 'Regression checking',
-        path: '/regression',
+        title: 'Getting started to use standalone Inspector',
+        path: '/implementation/start-using-inspector',
       },
       {
         type: 'route',
-        title: 'Data validation',
-        path: '/data-validation',
+        title: 'Getting started to use Inspector alongside Avo Functions',
+        path: '/implementation/start-using-inspector-with-avo-functions',
       },
       {
         type: 'route',
-        title: 'Mobile debuggers',
-        path: '/mobile-debuggers',
+        title:
+          'Getting started to implement tracking plan changes in code with Avo',
+        path: '/implementation/start-implementing-tracking-changes',
+      },
```

Comment:
This section feels quite heavy in the nav bar. Could we somehow reduce the length of the titles, or make this more visually easy to parse somehow else?

Reply:
Added suggestions for shorter titles, feel free to dismiss or edit!

Reply:
agreed